src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -279,6 +284,37 @@ public function addClassResource(\ReflectionClass $class)
     }
 
     /**
+     * Checks whether the requested class, interface or trait exists and registers the result for resource tracking.
+     *
+     * @param string $class
+     *
+     * @return bool
+     *
+     * @final
+     */
+    public function getClassExists($class)
```

Comment:
ok for a better name, but it should convey the fact that the return value has a meaning